scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/SimulationFacade.java

```diff
@@ -83,11 +68,11 @@ public Optional<Duration> getActivityDuration(ActivityInstance activityInstance)
       logger.error("You need to simulate before requesting activity duration");
       return Optional.empty();
     }
-    var duration = driver.getActivityDuration(planActInstanceIdToSimulationActInstanceId.get(activityInstance.getId()));
-    if(duration.isEmpty()){
-      logger.error("Incremental simulation is probably outdated, check that no activity is removed between simulation and querying");
-    }
-    return duration;
+     var duration = driver.getActivityDuration(planActInstanceIdToSimulationActInstanceId.get(activityInstance.getId()));
```

Comment:
[Minor:] Does the whitespace change here have significance?

Reply:
No, I am reverting this.